docs/enhancements/metallb-operator.md

```diff
@@ -0,0 +1,205 @@
+    # pool it will take an IP for this service
+    metallb.universe.tf/address-pool: metallb-api-vip
+  name: metallb-api
+  namespace: openshift-kube-apiserver
+spec:
+  ports:
+  - name: http
+    protocol: TCP
+    port: 6443
+    targetPort: 6443
+  selector:
+    app: openshift-kube-apiserver
+  type: LoadBalancer
+```
+
+## Proposal
+
+Adding an operator to Assisted Installer is a fairly simple task. This operator, however, requires a
+bit more input for Assisted Installer to be able to configure it properly. Specifcally, Assisted
+Installer needs to be able to answer the following questions:
+
+- Which is the *internal network*?
+- What is the IP on the **external network** to use for the API VIP?
+- What is the IP on the **external network** to use for the Ingress?
+- What routes (if any) should be used for the **external** API and Ingress endpoints?
+
+### Identifying Internal Network
+
+Assisted installer already has mechanims to know which is the internal network. Internally this
+network is called the `MachineNetwork`. In case of SNO, the `MachineNetwork` is required when defining
+the ACI. In the case of a multinode deployment, it is inferred from the API VIP whereas in the case
+of dual stack this information is required the same way as in the SNO case.
+
+Regardless, no changes should be required in assisted installer to identify the internal network.
+
+### Identifying External Network
+
+This enhancement proposes requesting the user for the following information in order to prepare the
+cluster for relocation:
+
+- External API VIP: This is an IP on the **external** network
+- External Ingress IP: This is an IP on the **external** network
+- External base domain: A domain to use for the **external** traffic (optional).
+
+Unlike for the **internal** network, this enhancement proposes to always infer the subnet of the
+external network from the External API VIP and the External Ingress IP provided by the user. There
+are utilities in Assisted Installer to do this already.
+
+Some requirements:
+
+- The **external** API VIP and Ingress IP must be on the same subnet.
+- The host must be connected to this network and the network must be routable
+- These 2 IPs must be optional. The IPAddressPool and Service won't be configured if this information is not provided
+- Either both are IPs are provided or none.
+- The *internal* network must be configured as a static network to prevent the MachineNetwork to
+change and, therefore, the IP of each of the nodes.
+
+### User Stories
+
+#### Story 1
+
+As a hardware provider, I want to be able to deliver an OCP appliance that can be shipped to a
+different location and reconnected.
+
+#### Story 2
+
+As a datacenter administrator, I want to be able to install an OCP cluster whose IP can be
+reconfigured as well as its domain.
+
+### Implementation Details/Notes/Constraints [optional]
+
+1. It's currently not possible to enable operators when using KubeAPI. Implementing this feature,
```

Comment:
ACM has its own ways of interacting with clusters which have been installed from it, including assisted clusters (e.g. using policies). Do you think you could utilize this to achieve the same?